content/matplotlib/concepts/pyplot/terms/bar/bar.md

```diff
@@ -0,0 +1,30 @@
+# .bar()
+
```

Comment:
You are missing an introductory paragraph explaining what the `.bar()` method does.